es/mappings/make.py

```diff
@@ -109,6 +117,120 @@
   CZECH: "czech",
 }
 
+SynonymGraphFilterImp = {
+            "type" : "synonym_graph",
+            "tokenizer": "keyword",
+            "synonyms" : [],
+}
+
+LanguageAnalyzerImp = {
+  ENGLISH: {
+      "english_synonym": {
+              "tokenizer":  "standard",
+              "filter": [
+                "english_possessive_stemmer",
+                "lowercase",
+                "english_stop",
+                "english_keywords",
+                "english_stemmer",
+                "synonym_graph"
+              ]
+        }
+  },
+  HEBREW: {
+    "hebrew_synonym": {
+            "tokenizer" : "standard",
+            "filter" : ["synonym_graph", "he_IL"],
+            "char_filter": [
+              "quotes"
+            ]
+    }
+  },
+  RUSSIAN: {
+     "russian_synonym": {
+            "tokenizer":  "standard",
+            "filter": [
+              "lowercase",
+              "russian_stop",
+              "russian_keywords",
+              "russian_stemmer",
+              "synonym_graph"
+            ]
+    }
+  },
+  SPANISH: {
+    "spanish_synonym": {
+            "tokenizer":  "standard",
+            "filter": [
+              "lowercase",
+              "spanish_stop",
+              "spanish_keywords",
+              "spanish_stemmer"
+            ]
+    },
+  }
+}
+
+LanguageFiltersImp ={
+  ENGLISH: {
+            "english_stop": {
+              "type":       "stop",
+              "stopwords":  "_english_" 
+            },
+            "english_keywords": {
+              "type":       "keyword_marker",
+              "keywords":   ["example"] 
```

Comment:
This example and in other languages looks like a mistake. Not sure why it is here.

Reply:
done